app/models/user.rb

```diff
@@ -1,4 +1,8 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```

Comment:
- [ ] As stated in the requirements, let kindly make sure that the app asks for confirmation of email during the process of signing up hence `:confirmation` should be enabled in the user model as below

```.ruby
     devise :database_authenticatable, :registerable,
				          :recoverable, :rememberable, :validatable, :confirmable
```